modules/monitoring-moving-monitoring-components-to-different-nodes.adoc

```diff
@@ -3,112 +3,102 @@
 // * observability/monitoring/configuring-the-monitoring-stack.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="moving-monitoring-components-to-different-nodes_{context}"]
-= Moving monitoring components to different nodes
+// tag::CPM[]
+[id="moving-monitoring-components-to-different-nodes-cpm_{context}"]
+= Moving monitoring components to different nodes for core platform monitoring
+// end::CPM[]
 
-ifndef::openshift-dedicated,openshift-rosa[]
-To specify the nodes in your cluster on which monitoring stack components will run, configure the `nodeSelector` constraint in the component's `ConfigMap` object to match labels assigned to the nodes.
+// tag::UWM[]
+[id="moving-monitoring-components-to-different-nodes-uwm_{context}"]
+= Moving monitoring components to different nodes for monitoring for user-defined projects
+// end::UWM[]
+
+// Set attributes to distinguish between cluster monitoring examples and user workload monitoring examples.
+// tag::CPM[]
+:configmap-name: cluster-monitoring-config
+:namespace-name: openshift-monitoring
+// end::CPM[]
+// tag::UWM[]
+:configmap-name: user-workload-monitoring-config
+:namespace-name: openshift-user-workload-monitoring
+// end::UWM[]
+
+// tag::CPM[]
+To specify the nodes in your cluster on which monitoring stack components will run, configure the `nodeSelector` constraint for the components in the `cluster-monitoring-config` config map to match labels assigned to the nodes.
 
 [NOTE]
 ====
 You cannot add a node selector constraint directly to an existing scheduled pod.
 ====
-endif::openshift-dedicated,openshift-rosa[]
+// end::CPM[]
 
-ifdef::openshift-dedicated,openshift-rosa[]
-You can move any of the components that monitor workloads for user-defined projects to specific worker nodes. It is not permitted to move components to control plane or infrastructure nodes.
-endif::openshift-dedicated,openshift-rosa[]
+// tag::UWM[]
+You can move any of the components that monitor workloads for user-defined projects to specific worker nodes. 
+
+[WARNING]
+====
+It is not permitted to move components to control plane or infrastructure nodes.
+====
+// end::UWM[]
 
 .Prerequisites
+
+// tag::CPM[]
+* You have access to the cluster as a user with the `cluster-admin` cluster role.
```

Comment:
The [guidelines say that "wildcards" aren't supported](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#including-by-tags) - I wasn't sure if that applies here, so I guess I would just check and make sure that they look OK after they get through PV1.

Reply:
Hello @kcarmichael08 thank you for your questions, I am learning about this all too! :slightly_smiling_face: The guide mentions the support for only single wildcard, which is single asterisk, so I used double, but I am also not 100% sure if that support is the same. However, it still says that we can use it if it builds correctly, s I made sure that the builds are as they should be :slightly_smiling_face: